docs/source/compiler.rst

```diff
@@ -325,25 +398,161 @@ rewiring, which is controlled by ``PRAGMA INITIAL_REWIRING``. The syntax is as f
    PRAGMA INITIAL_REWIRING "<type>"
 
 Including this `before any non-pragmas` will allow the compiler to alter its rewiring
-behavior. The possible options are:
-
-+ ``PARTIAL`` (default): The compiler will start with nothing assigned to each
-  physical qubit. Then, it will fill in the logical-to-physical mapping as it
-  encounters new qubits in the program, making its best guess for where they
-  should be placed.
-+ ``NAIVE``: The compiler will start with an identity mapping as the initial
-  rewiring.  In particular, qubits will **not** be rewired unless the program
-  requests a qubit-qubit interaction not natively available on the QPU.
-+ ``RANDOM``: the compiler will start with a random permutation.
-+ ``GREEDY``: the compiler will make a guess for the initial rewiring based on a
-  quick initial scan of the entire program.
+behavior.
+
+The default initial rewiring strategy
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 .. note::
-   The default option, ``PARTIAL``, may lead to higher-performing programs because the compiler has
-   more freedom to optimize the layout of the gates on the qubits. ``NAIVE``, a simpler option,
-   follows the "Do What I Mean" (DWIM) principle. It is the least sophisticated, but attempts to
-   follow what the user has constructed with their program.
-  
+
+   Each initial rewiring strategy is described in more detail after the discussion about defaults.
```

Comment:
We might want a "strategy" heading with an intro, listing the options as bullets and noting what the purpose of a strategy is (and importantly, is not), and what tradeoffs it's working with. Then we can mention this default and we wouldn't need this note to explain the flow.

Reply:
Could also include a brief "which situations would suggest which strategy" comparison, like you have under `GREEDY` now.